src/Zynga/Framework/Lockable/Cache/V1/Driver/Caching.hh

```diff
@@ -58,38 +58,6 @@ class Caching extends FactoryDriverBase implements DriverInterface {
       throw $e;
     }
   }
-  
```

Comment:
Reason for taking this out is that let dev's use lock & unlock directly instead of depending on isLocked check, this approach avoids messing the locking issues around diff thread.